docs/user/gcp/install_upi.md

```diff
@@ -84,10 +99,20 @@ data["spec"]["mastersSchedulable"] = False;
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
+```console
+spec:
+  mastersSchedulable: false
+  policy:
+    name: ''
+```
+
 ### Remove DNS Zones (Optional)
 If you don't want [the ingress operator][ingress-operator] to create DNS records on your behalf, remove the `privateZone` and `publicZone` sections from the DNS configuration.
 If you do so, you'll need to [add ingress DNS records manually](#add-the-ingress-dns-records) later on.
 
+If you are installing into a [Shared VPC][sharedvpc],
+remove these sections.
```

Comment:
nit: `these` should probably be expanded on..

Reply:
Resolved in 657a91d789da80eb9e2e96efc4ded29f490d64a9